docs/use-transition.md

```diff
@@ -63,7 +63,7 @@ Here is a typical dropdown component from Tailwind.
 <div class="relative"
      data-controller="transition click-outside"
      data-transition-target="content"
-     data-action="click-outside:click:ouside->">
+     data-action="click-outside:click:outside->transition#leave">
```

Comment:
Please let me know if this was *not* a typo - and I am unaware of some magic syntax :)